samples/snippets/csharp/VS_Snippets_CFX/s_ueenvelopeversion/cs/program.cs

```diff
@@ -12,76 +12,71 @@ class Program
-     string[] ultDestsS12 = envS12.GetUltimateDestinationActorValues();
-     // </Snippet5>
-     
-     string ultS12 = ultDestsS12[1];
-
-     // <Snippet6>
-     string toStrS12 = envS12.ToString();
-     // </Snippet6>
-     
-     EnvelopeVersion envNone = EnvelopeVersion.None;
-     string nextDestNone = envNone.NextDestinationActorValue;
-     //The following code throws a System.ArgumentReferenceExeption.
-     //The object reference is not set to an instance of an object
-     // string[] ultDestsNone = envNone.GetUltimateDestinationActorValues();
-     string toStrNone = envNone.ToString();
-
-     //EnvelopeVersions
-     Console.WriteLine("EnvelopeVersion.Soap11: {0}", envS11);
-     Console.WriteLine("EnvelopeVersion.Soap12: {0}", envS12);
-     Console.WriteLine("EnvelopeVersion.None: {0}", envNone);
-     Console.WriteLine();
-
-     //NextDestination
-     Console.WriteLine("NextDest EnvelopeVersion.Soap11: {0}", nextDestS11);
-     Console.WriteLine("NextDest EnvelopeVersion.Soap12: {0}", nextDestS12);
-     Console.WriteLine("NextDest EnvelopeVersion.None: {0}", nextDestNone);
-     Console.WriteLine();
-
-     //UltimateDestinations
-     Console.WriteLine("UltDest EnvelopeVersion.Soap11: {0}", ultS11);
-     Console.WriteLine("UltDest EnvelopeVersion.Soap12: {0}", ultS12);
-     //Console.WriteLine("UltDest EnvelopeVersion.None: {0}", ultDestsNone);
-     Console.WriteLine();
-
-     //ToString
-     Console.WriteLine("EnvelopeVersion.Soap11.ToString(): {0}", toStrS11);
-     Console.WriteLine("EnvelopeVersion.Soap11.ToString(): {0}", toStrS12);
-      Console.WriteLine("EnvelopeVersion.Soap11.ToString(): {0}", toStrNone);
-      Console.WriteLine();
-
+            // MessageVersion
+            // <Snippet1>
+            EnvelopeVersion envS11 = EnvelopeVersion.Soap11;
+            // </Snippet1>
+            string nextDestS11 = envS11.NextDestinationActorValue;
+            string[] ultDestsS11 = envS11.GetUltimateDestinationActorValues();
+            string ultS11 = ultDestsS11[0];
+            string toStrS11 = envS11.ToString();
+
+            // <Snippet2>
+            EnvelopeVersion envS12 = EnvelopeVersion.Soap12;
+            // </Snippet2>
+
+            // <Snippet3>
+            EnvelopeVersion envNotSOAP =  EnvelopeVersion.None;
+            // </Snippet3>
+
+            // <Snippet4>
+            string nextDestS12 = envS12.NextDestinationActorValue;
+            // </Snippet4>
+
+            // <Snippet5>
+            string[] ultDestsS12 = envS12.GetUltimateDestinationActorValues();
+            // </Snippet5>
+             
+            string ultS12 = ultDestsS12[1];
+
+            // <Snippet6>
+            string toStrS12 = envS12.ToString();
+            // </Snippet6>
+             
+            EnvelopeVersion envNone = EnvelopeVersion.None;
+            string nextDestNone = envNone.NextDestinationActorValue;
+            //The following code throws a System.ArgumentReferenceException.
```

Comment:
The GitHub diff is unhelpful here! The single typo I corrected was `ArgumentReferenceExeption`  > `ArgumentReferenceException` - the rest of the change is indentation.